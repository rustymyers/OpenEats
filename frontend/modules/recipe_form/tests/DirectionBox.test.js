import React from 'react';
import DirectionBox from '../components/DirectionBox';
import createComponentWithIntl from '../../../jest_mocks/createComponentWithIntl';

test('DirectionBox component test', () => {
  const component = createComponentWithIntl(
    <DirectionBox/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
