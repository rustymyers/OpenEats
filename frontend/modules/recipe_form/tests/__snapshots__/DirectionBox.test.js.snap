// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DirectionBox component test 1`] = `
<div
  className="live-editor"
>
  <ul
    className="nav nav-tabs"
  >
    <li
      className="left active"
    >
      <span>
        Â 
        <span
          className="glyphicon glyphicon-info-sign"
          onClick={null}
          onMouseOut={[Function]}
          onMouseOver={[Function]}
        />
      </span>
    </li>
    <li
      className="active"
    >
      <span
        onClick={[Function]}
      >
        Editor
      </span>
    </li>
    <li
      className=""
    >
      <span
        onClick={[Function]}
      >
        Preview
      </span>
    </li>
  </ul>
  <div
    className="content"
  >
    <div
      className={undefined}
    >
      <div
        className="form-group null"
      >
        <textarea
          className="form-control"
          name={undefined}
          onChange={[Function]}
          placeholder={undefined}
          rows="8"
          type={undefined}
          value=""
        />
      </div>
    </div>
  </div>
  <div
    className="help-inline"
  />
</div>
`;
